content/docs/1.4.0/volumes-and-nodes/storage-tags.md

```diff
@@ -28,10 +28,19 @@ When creating a volume, specify the disk tag and node tag in the UI.
 
 ### Kubernetes
 
-Use Kubernetes StorageClass setting to specify tags.
+Use Kubernetes StorageClass setting to specify tags to schedule replicas only on the matching nodes.
```

Comment:
NIT: `setting` -> `option`/`parameter`
Similar below

Reply:
Got it, I replaced the word `setting` with `parameter` or `field` to avoid misunderstandings.

In addition, I created a ticket ([#4776](https://github.com/longhorn/longhorn/issues/4776)) to document the storage class parameters and how to modify them.

Reply:
OK. You can describe each parameters of the StorageClass in the new ticket. But it's better to mention how to add `nodeSelector` for the **existing** default StorageClass here as I mentioned.

Reply:
Got it. I will add the way with using `nodeSelector` field in this page. Thank you.

Reply:
@shuo-wu  I've added an example of `nodeSelector` to try to make it clear how to use this parameter. Thank you.